web/themes/custom/pineapple/sass/core/01_variables/_typography.scss

```diff
@@ -4,19 +4,31 @@
 
 // Import Google Fonts
 @import url('https://fonts.googleapis.com/css?family=Fredoka+One');
-@import url('https://fonts.googleapis.com/css?family=Montserrat:100,200,300,400,500,600,700,900');
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
 
 /// The base font size for the site.
 $base-font-size:    16px;
 /// The base line height for the site.
 $base-line-height:  1.6rem;
 
 /// The base font family for the site.
-$base-font-family: 'Montserrat', sans-serif;
+$base-font-family: 'Lato', sans-serif;
+@font-face {
+  font-family: ‘Lato’;
+  src: url('../../../fonts/Lato/Lato-Regular.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
```

Comment:
You're only adding one font but the fonts folder has loads of fonts, either add some more here or remove some from the fonts folder

Reply:
done